src/hb-ot-math.cc

```diff
@@ -101,8 +110,11 @@ hb_ot_math_get_glyph_italics_correction (hb_font_t *font,
 
 /**
  * hb_ot_math_get_glyph_top_accent_attachment:
- * @font: #hb_font_t from which to retrieve the value
- * @glyph: glyph index from which to retrieve the value
+ * @font: #hb_font_t to work upon
+ * @glyph: The glyph index from which to retrieve the value
+ *
+ * Fetches a top-accent-attachment value (if one exists) for the specified
+ * glyph index.
  *
  * Return value: the top accent attachment of the glyph or 0
```

Comment:
For glyphs that are not covered by the `MathTopAccentAttachment` (or in case it is or the `MATH` table are missing) this function actually returns glyph advance width / 2, so zero is a valid value not an indicator the glyph is not covered.